data/sql/updates/pending_db_world/rev_1641437831789184769.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1641437831789184769');
+
+DELETE FROM `smart_scripts` WHERE `entryorguid`=2362200 AND `source_type`=9 AND `id`=2 AND `link`=0;
```

Comment:
we have 5 ids for this entry (0 to 5), removing just id 2 will break the id chain. please adjust

Reply:
Oh yeah, didn't think about it. Will fix it. Thank u.